tests/gordo_components/workflow/test_server_to_sql/test_server_to_sql.py

```diff
@@ -0,0 +1 @@
+# TODO: Add relevant tests
```

Comment:
:laughing:

Reply:
This was because there was a single test in here, but the function it tested was removed from refactoring, and no other functions in `server_to_sql` had tests to begin with.